config/wpcalypso.json

```diff
@@ -67,7 +67,6 @@
 		"upgrades/domain-management/site-redirect": true,
 		"upgrades/domain-management/transfer": true,
 		"upgrades/domain-search": true,
-	  	"upgrades/free-trials": true,
```

Comment:
We should not forget to check again for uses of this feature flag just before merging - we're about to introduce a new one in https://github.com/Automattic/wp-calypso/pull/2113.